lib/karafka/base_consumer.rb

```diff
@@ -41,14 +41,14 @@ def params_batch=(messages)
       @params_batch = Karafka::Params::ParamsBatch.new(messages, topic.parser)
     end
 
-    # Executes the default controller flow.
+    # Executes the default consumer flow.
     def call
       process
     end
 
     private
 
-    # We make it private as it should be accesible only from the inside of a controller
+    # We make it private as it should be accesible only from the inside of a consumer
```

Comment:
We make it private as it should be accessible only from the inside of a consumer